hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ConfigurationService.java

```diff
@@ -48,6 +48,8 @@
 @SuppressWarnings("checkstyle:methodcount")
 public interface ConfigurationService {
 
+    String SERVICE_NAME = "configurationService";
```

Comment:
We have this: `ClusterWideConfigurationService.SERVICE_NAME`. I would use that in `MessageTask`s and `Operation`s. WDYT?